gulpfile.js

```diff
@@ -163,7 +160,8 @@ gulp.task('cloc', ['build'], function() {
     //If running cloc on source succeeded, also run it on the tests.
     return source.then(function() {
         return new Promise(function(resolve, reject) {
-            cmdLine = 'perl ' + clocPath + ' --quiet --progress-rate=0 --read-lang-def=' + cloc_definitions + ' Specs/';
+            cmdLine = 'perl ' + clocPath + ' --quiet --progress-rate=0' +
```

Comment:
Do we want to exclude .eslintrc.json and the karma configurations?

Reply:
I wouldn't, since it's all "code".

Reply:
We were counting them before so I think it's okay to leave as is